frontend/packages/console-app/src/components/guided-tours/TourItemHeader.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { Status } from '@console/shared';
+import { Badge } from '@patternfly/react-core';
+import { ClockIcon } from '@patternfly/react-icons';
+import { GuidedTourStatus } from './utils/guided-tour-status';
+import './TourItemHeader.scss';
+
+type TourItemHeaderProps = {
+  status: string;
+  duration: number;
+  name: string;
+};
+const TourItemHeader: React.FC<TourItemHeaderProps> = ({ status, duration, name }) => (
+  <div className="odc-tour-item-header">
+    {name}
+    <div className="odc-tour-item-header__status">
+      {/* fix me - change badges to labels once migrated to PF4 - */}
+      {status !== GuidedTourStatus.NOT_STARTED && (
+        <Badge isRead key={1} className="odc-tour-item-header--rightmargin">
+          <Status status={status} />
```

Comment:
Missing inprogress icon:
![image](https://user-images.githubusercontent.com/14068621/85782538-a3868900-b6f4-11ea-8110-a1ad54a2b528.png)
vs
![image](https://user-images.githubusercontent.com/14068621/85782592-abdec400-b6f4-11ea-96b1-73cadcbe98f6.png)

Ignoring the color for now.

Reply:
Updated `In progress` -> to `In Progress`